scripts/add-icon-data.js

```diff
@@ -88,85 +89,103 @@ const getIconDataFromAnswers = (answers) => ({
 
 const answers = {};
 
-answers.title = await input({
-  message: 'Title',
-  validate: titleValidator,
-});
-
-answers.hex = await input({
-  message: 'Hex',
-  validate: hexValidator,
-  transformer: hexTransformer,
-});
-
-answers.source = await input({
-  message: 'Source',
-  validate: sourceValidator,
-});
+try {
+  answers.title = await input({
+    message: 'Title:',
+    validate: titleValidator,
+  });
 
-answers.hasGuidelines = await confirm({
-  message: 'The icon has brand guidelines?',
-});
+  answers.hex = await input({
+    message: 'Hex:',
+    validate: hexValidator,
+    transformer: hexTransformer,
+  });
 
-if (answers.hasGuidelines) {
-  answers.guidelines = await input({
-    message: 'Guidelines',
+  answers.source = await input({
+    message: 'Source URL:',
     validate: sourceValidator,
   });
-}
-
-answers.hasLicense = await confirm({
-  message: 'The icon has brand license?',
-});
 
-if (answers.hasLicense) {
-  answers.licenseType = await input({
-    message: 'License type',
-    validate: (text) => Boolean(text),
+  answers.hasGuidelines = await confirm({
+    message: 'The icon has brand guidelines?',
   });
 
-  answers.licenseUrl = await input({
-    message: 'License URL' + chalk.reset(' (optional)'),
-    validate: (text) => !Boolean(text) || sourceValidator(text),
+  if (answers.hasGuidelines) {
+    answers.guidelines = await input({
+      message: 'Guidelines URL:',
+      validate: sourceValidator,
+    });
+  }
+
+  answers.hasLicense = await confirm({
+    message: 'The icon has brand license?',
   });
-}
 
-answers.hasAliases = await confirm({
-  message: 'This icon has brands aliases?',
-  default: false,
-});
+  if (answers.hasLicense) {
+    const licenseTypes =
+      jsonSchema.definitions.brand.properties.license.oneOf[0].properties.type.enum.map(
+        (license) => {
+          return { value: license };
+        },
+      );
+    answers.licenseType = await autocomplete({
+      message: 'License type:',
+      source: async (input) => {
+        input = (input || '').trim();
+        return Boolean(input)
+          ? licenseTypes.filter((license) =>
+              license.value.toLowerCase().includes(input.toLowerCase()),
+            )
+          : licenseTypes;
```

Comment:
You don't need a `Boolean` casting here:
```suggestion
        input = (input || '').trim();
        return input
          ? licenseTypes
          : licenseTypes.filter((license) =>
              license.value.toLowerCase().includes(input.toLowerCase()),
            );
```